### ISSUE TITLE
How the autoencoder in the example project bind input data as the label data

### ISSUE THREAD

**Issue by gleery** [dmlc/mxnet#1859]
I've read the code in the autoencoder, but I couldn't figure out how it binds the input data as the label data. The autoencoder needs to share weights between stacks and finetuned net, so the training process is done manually with codes, this example is more complicated than others, hope there could be a readme file to help understanding.
@piiswrong

**Comment by piiswrong** [comment]
there is no difference between data/label in executor. The difference is in FeedForward model. If you are writing your own training loop you can use the same input as data and label

**Comment by gleery** [comment]
I figure it out, it binds the label data when it construct the net, guess I just miss it.
